src/rust/engine/process_execution/src/local.rs

```diff
@@ -45,6 +45,7 @@ pub struct CommandRunner {
   named_caches: NamedCaches,
   cleanup_local_dirs: bool,
   platform: Platform,
+  spawn_lock: Arc<RwLock<bool>>,
```

Comment:
```suggestion
  spawn_lock: Arc<RwLock<()>>,
```

To avoid people needing to think about what the `bool` may represent :)

Reply:
Fixed.